.github/workflows/maven.yml

```diff
@@ -0,0 +1,44 @@
+ # Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This should be in a separate PR as it's not directly related to fixing this bug

Reply:
I am not a GitHub expert at all. I can remove maven.yml in my repo, no problem. Is it enough for this PR ?

Reply:
you need to push too.